client/members.go

```diff
@@ -255,6 +277,14 @@ func assertStatusCode(got int, want ...int) (err error) {
 	return fmt.Errorf("unexpected status code %d", got)
 }
 
+type membersAPIActionLeader struct{}
+
+func (l *membersAPIActionLeader) HTTPRequest(ep url.URL) *http.Request {
+	u := v2MembersURL(ep)
+	req, _ := http.NewRequest("GET", u.String()+"/leader", nil)
```

Comment:
Please follow the style of the other functions: https://github.com/coreos/etcd/blob/eaaf98348ce433ff0a8ff94552649af215754df3/client/members.go#L216
I also suggest splitting out `const leaderSuffix = "leader"`.